release-20.2: ui: Replace node-sass with sass

nathanstilwell opened the pull request:
Backport 1/1 commits from #65971.

/cc @cockroachdb/release

---

Replaced node-sass (a nodejs binding for LibSass written in C++) with
sass (a Sass compiler written in Dart and compiled to pure JavaScript).
Also upgraded sass-loader (a Sass loader for Webpack) to a version that
prefers sass over node-sass (meaning no Webpack changes were required).

Associated `yarn-vendored` change: https://github.com/cockroachdb/yarn-vendored/pull/76

cockroach-teamcity commented:
<!-- Reviewable:start -->
This change is [<img src="https://reviewable.io/review_button.png" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/cockroachdb/cockroach/66009)
<!-- Reviewable:end -->

Commit:
ui: Replace node-sass with sass

Replaced node-sass (a nodejs binding for LibSass written in C++) with
sass (a Sass compiler written in Dart and compiled to pure JavaScript).
Also upgraded sass-loader (a Sass loader for Webpack) to a version that
prefers sass over node-sass (meaning no Webpack changes were required).

Release note: None

koorosh commented:
@nathanstilwell , just realized that I didn't merge my previous PR (https://github.com/cockroachdb/yarn-vendored/pull/74) into master so this build misses some of packages. Sorry for this, I've already merged it, so could you bump to `yarn-vendor` master branch to fix build failure.

koorosh reviewed (approved):
LGTM (with only one comment above)
